examples/rust/tcp_inlet_and_outlet/Cargo.toml

```diff
@@ -9,6 +9,7 @@ rust-version = "1.56.0"
 
 [dependencies]
 ockam = { path = "../../../implementations/rust/ockam/ockam" }
+portpicker = "0.1.1"
```

Comment:
Portpicker was last contributed to 3years ago, which means its unmaintained, can we instead copy some of the functions to the example code?

Reply:
Oh ok,
I didn't see that...
I will search for some alternatives otherwise I will see which functions can be used from `portpicker`

Reply:
We have examples of how to do this without any library. Check [this](https://github.com/build-trust/ockam/blob/18929d337638ab54f063ce2c84b1a5750762cd54/implementations/rust/ockam/ockam_command/src/util/mod.rs#L123) out as an example.

Reply:
Thank you @adrianbenavides 
I used the soln you provided.

Resolved